cypress/integration/26-register_registration/creator_navigates_to_resource_type_tab/navigate_to_resource_type.js

```diff
@@ -0,0 +1,276 @@
+});
+
+//   Scenario: Creator sees that fields are validated on Resource Type tab
+
+//   @393
+//   Scenario: Creator navigates to the Resource Type tab and selects Resource type "Report"
+And('they select the Resource type "Report"', () => {
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-context-type-Report]').click({ force: true });
+});
+//   @392
+//   Scenario: Creator navigates to the Resource Type tab and selects Resource subtype "Anthology"
+And('they select Resource subtype "Anthology" from the list', () => {
+  cy.get('[data-testid=publication-instance-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-instance-type-BookAnthology]').click({ force: true });
+});
+And('they see fields for', (dataTable) => {
+  cy.testDataTestidList(dataTable, BOOK_FIELDS);
+});
+//   | ISBN                  |
+//   | Total number of pages |
+//   | NPI discipline        |
+And('they see a Search box for "Title of the Series"', () => {
+  cy.get('[data-testid=series-search-field]').should('be.visible');
+});
+And('they see a preselected value for Peer review "Not peer reviewed"', () => {
+  cy.get('[data-testid=peer_review-false]').within((checkbox) => {
+    cy.wrap(checkbox).get('input').should('be.checked');
+  });
+});
+
+// @394
+// Scenario: Creator navigates to the Resource Type tab and selects Resource type "Student thesis"
+And('they select the Resource type "Student thesis"', () => {
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-context-type-Degree]').click({ force: true });
+});
+//   | Bachelor thesis      |
+//   | Master thesis        |
+//   | Doctoral thesis      |
+//   | Other student thesis |
+
+// @395
+// Scenario: Creator sees fields for Resource subtype "Chapter in book"
+// | Part of book/report |
+And('they select the Registration Subtype "Chapter in book"', () => {
+  cy.get('[data-testid=publication-instance-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-instance-type-ChapterArticle]').click({ force: true });
+  cy.wrap('Chapter in book').as('subType');
+});
+Then('they see an information box describing that a Container book must be published first', () => {
+  cy.get('[data-testid=info-anthology]').should('be.visible');
+});
+//   | DOI                            |
+//   | Search box for published books |
+//   | Pages from                     |
+//   | Pages to                       |
+//   | Peer reviewed                  |
+
+// @1409
+// Scenario Outline: Creator selects Contribution to Journal and Peer Review Details are hidden
+And('they select type Contribution to Journal', () => {
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-context-type-Journal]').click({ force: true });
+});

```

Comment:
mangler en `When` her som tester de forskjellige subtypene som examples refererer til?

Reply:
Den ligger i `test_subtypes.js` Merk at `When` og `And` er ekvivalenter i praksis.